chart/templates/configmaps/statsd-configmap.yaml

```
@@ -29,9 +29,13 @@ metadata:
     release: {{ .Release.Name }}
     chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
     heritage: {{ .Release.Service }}
-    {{- with .Values.labels }}
-      {{- toYaml . | nindent 4 }}
-    {{- end }}
+{{- with .Values.labels }}
+{{ toYaml . | indent 4 }}
+{{- end }}
```

Why did you change this?
```suggestion
    {{- with .Values.labels }}
      {{- toYaml . | nindent 4 }}
    {{- end }}
```